exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/project/ProjectMemoryManager.java

```diff
@@ -301,4 +323,21 @@ public void update() {
         logger.debug("BATCH_STATS, incoming: {}", getRecordBatchSizer());
         updateIncomingStats();
     }
+
+    public static int getMetadataWidth(ValueVector vv) {
+        int width = 0;
+        if (vv instanceof NullableVector) {
+            width += ((NullableVector)vv).getBitsVector().getPayloadByteCount(1);
+        }
+
+        if (vv instanceof VariableWidthVector) {
+            width += ((VariableWidthVector)vv).getOffsetVector().getPayloadByteCount(1);
+        }
+
+        if (vv instanceof BaseRepeatedValueVector) {
```

Comment:
Should this class be changed to the interface "RepeatedValueVector" to cover repeated maps and lists as well ?

Reply:
Yes. I missed it. Thanks for noticing it.

Reply:
Sorry for misleading -- thanks for the fix .....